tests/parallel_test.go

```diff
@@ -13,7 +13,7 @@ import (
 func TestSelectWhileSyncing(t *testing.T) {
 	runOneToOneTestClientWithAuth(t, defaultServerOptions(t), func(client *client.Client, s *testSession) {
 		// Define some mailbox names.
-		mailboxNames := []string{"A", "B", "C", "D", "E", "F", "G", "H", "I", "J"}
+		mailboxNames := []string{"Apply", "B", "C", "D", "E", "F", "G", "H", "I", "J"}
```

Comment:
Ah so maybe the `A` -> `Apply` in the test expectation came from here, more find-replace shenanigans? or was this intentional for some reason?

Reply:
Fixed.